### DISCUSSION TITLE
Create a qcow2 image as well

### DISCUSSION THREAD

**Discussion by thibaultmol** [nextcloud/all-in-one#5315] [Ideas] [answered]
Besides the OVA image it might be good to provide a qcow2 format as well (Proxmox, which is one of the biggest virtualization platforms. using QEMU. uses this format instead of OVA)

**Comment by szaimen** [comment, marked as answer]
Hi, I fear this is out of scope. 
We are already struggling keeping the existing ones up-to-date: https://github.com/nextcloud/all-in-one/issues/4775

However, feel free to build your own.

**Comment by theoo** [comment]
An .OVA file can be easily converted to a .qcow2 file.

That said, I completely agree with the original request—OpenStack and KVM users specifically need a .qcow2 file. These are the leading open-source virtualization solutions, so it makes far more sense to provide this format rather than VMware or Hyper-V, both of which are proprietary offerings from Big Tech. After all, we’re talking about Nextcloud, a project built to challenge the current monopoly.
